src/plugins/intel_cpu/src/nodes/executors/common/ref_convert.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "nodes/executors/convert.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class CommonConvertExecutor : public ConvertExecutor {
+public:
+    explicit CommonConvertExecutor(const ExecutorContext::CPtr context);
```

Comment:
This constructor may be inherited from the base class.

Reply:
fixed